src/reducers/modules.js

```diff
@@ -5,13 +5,20 @@ const initialState = [];
 
 export default (state = initialState, action) => {
   let newState = state;
+
   switch (action.type) {
+    case 'NEW_MODULE':
+      return [...newState, {name: 'New Module', remarks: 'New Remarks', states: {Initial: {type: 'Initial'}}}]
     case 'EDIT_NODE':
       let path = action.data.path.join('.');
       let value = Object.values(action.data.update).map((v) => v.id)[0]
       newState = {...state};
       _.set(newState, path, value);
       return {...newState}
+    case 'ADD_NODE':
+      debugger
```

Comment:
For now it's not an issue, but should I should probably set up a linter rule to catch errant `debugger` statements.

Reply:
Ewww yeah.  Pet peeve of mine is people that commit debugger (and binding.pry).  I'll fix.